apps/web/src/components/molecules/InterestGroupListItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import OnlineIcon from "@/components/atoms/OnlineIcon";
+import { InterestGroup } from "@dotkomonline/types";
+import Link from "next/link";
+import { FC } from "react";
+
+export interface InterestGroupListItemProps {
+    interestGroup: InterestGroup;
+}
+
+const InterestGroupListItem: FC<InterestGroupListItemProps> = (
+    props: InterestGroupListItemProps
+) => (
+    <div className="p-4 py-8 text-center m-1 shadow-md rounded-lg border-slate-3 border min-w-[250px] flex flex-col">
+        <OnlineIcon className="w-5/12 mx-auto max-w-[150px] min-w-[120px]" />
+        <h2 className="!text-3xl border-none !mt-4">
+            {props.interestGroup.name}
+            {!props.interestGroup.isActive && <> [inaktiv]</>}
```

Comment:
```suggestion
            {!props.interestGroup.isActive && <span> [inaktiv]<span/>}
```
Dette var vanskelig å lese